src/util/MongoBinary.js

```diff
@@ -23,10 +24,13 @@ export default class MongoBinary {
   static cache: MongoBinaryCache = {};
 
   static async getPath(opts?: MongoBinaryOpts = {}): Promise<string> {
+    const legacyDLDir = path.resolve(os.homedir(), '.mongodb-binaries');
     const defaultOptions = {
       downloadDir:
         process.env?.MONGOMS_DOWNLOAD_DIR ||
-        path.resolve(findCacheDir({ name: 'mongodb-memory-server' }), 'mongodb-binaries'),
+        (fs.existsSync(legacyDLDir)
```

Comment:
Why did you add this check ?
And not just piped everything like this : 
```javascript
downloadDir:
        process.env?.MONGOMS_DOWNLOAD_DIR ||
        path.resolve(os.homedir(), '.mongodb-binaries') ||
        path.resolve(findCacheDir({ name: 'mongodb-memory-server' }), 'mongodb-binaries')
```

Reply:
`path.resolve(os.homedir(), '.mongodb-binaries')`
is always truthy (it just returns a path), so

`path.resolve(findCacheDir({ name: 'mongodb-memory-server' }), 'mongodb-binaries')`
would never be used